src/client/Client.cc

```diff
@@ -9923,20 +9950,17 @@ int Client::ll_create(Inode *parent, const char *name, mode_t mode,
 	        0, 0, 0, NULL, &created, uid, gid);
     if (r < 0)
       goto out;
-
-    if ((!in) && fhp)
-      in = (*fhp)->inode;
```

Comment:
On Mon, Jun 1, 2015 at 7:22 AM, Gregory Farnum <notifications@github.com>
wrote:

> In src/client/Client.cc
> <https://github.com/ceph/ceph/pull/4753#discussion_r31395431>:
>
> > @@ -9923,20 +9950,17 @@ int Client::ll_create(Inode *parent, const char *name, mode_t mode,
> >  	        0, 0, 0, NULL, &created, uid, gid);
> >      if (r < 0)
> >        goto out;
> > -
> > -    if ((!in) && fhp)
> > -      in = (*fhp)->inode;
>
>
check Client::_create, you can sess that ((!in) && fhp) never be true

> Why don't we need this logic any more?
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/ceph/ceph/pull/4753/files#r31395431>.
>